daml-script/dump/src/main/scala/com/daml/script/dump/Config.scala

```diff
@@ -19,6 +23,15 @@ object Config {
   def parse(args: Array[String]): Option[Config] =
     parser.parse(args, Empty)
 
+  private def parseLedgerOffset(s: String): LedgerOffset = LedgerOffset {
```

Comment:
I assume you checked that we don’t have this anywhere else? I think extractor accepts offsets as well but from a quick look it seems like it doesn’t quite have the need for this (it only accepts absolute offsets).